CHANGELOG.md

```diff
@@ -28,3 +28,5 @@
 - fix: update RPC URL to use localhost instead of 0.0.0.0 in hurl.config file
 - fix: update the default port for running Madara locally in getting-started.md
   file from 9933 to 9944.
+- dev: replace the 0 initial gas value with u128::MAX because view call
```

Comment:
Is this OK for a long time?
Shouldn't the value be something else than u128::MAX?

Reply:
@tdelabro We discussed that this is customizable for appchains, but we leave this value to be u128::MAX. We don't know which contracts to benchmark to get a different value.